doc/api/https.md

```diff
@@ -7,47 +7,53 @@ separate module.
 
 ## Class: https.Agent
 <!-- YAML
-added: v0.4.5
+added: v0.4.4
 -->
 
-An Agent object for HTTPS similar to [`http.Agent`][].  See [`https.request()`][]
-for more information.
+**Inherits:** [`http.Agent`]
+
+An Agent object for HTTPS.
 
 ## Class: https.Server
 <!-- YAML
 added: v0.3.4
 -->
 
-This class is a subclass of `tls.Server` and emits events same as
-[`http.Server`][]. See [`http.Server`][] for more information.
+**Inherits:** [`tls.Server`]
+**Implements:** [`http.Server`]
```

Comment:
I've been considering this also but it's going to be a bit difficult given how we currently manage the documentation. I've been thinking about a more radical approach where we break the documentation up a bit more in order to make individual APIs easier to document and modify. I think, for now, however, this is a good change as is.